samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/Package.appxmanifest

```diff
@@ -45,6 +45,5 @@
 
   <Capabilities>
     <Capability Name="internetClient" />
-	<DeviceCapability Name="location" />
```

Comment:
><DeviceCapability Name="location" /> [](start = 1, length = 36)

Probably need this. Did you check out WOCCatalog's location features to see if they work... correctly?